content/es/kubernetes.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Kubernetes
+status: Completed
+category: technology
+---
+
+## Qué es?
+Kubernetes, comúnmente abreviado como K8s, es una herramienta de código abierto para la automatización de infraestructura moderna. Puedes compararlo con un sistema operativo en un Centro de Datos que administra las aplicaciones ejecutándose en [sistemas distribuidos](/sistemas_distribuidos/) (ó como el sistema operativo en tu laptop que administra tus aplicaciones.)
+
+Kubernetes administra [contenedores](/contenedores/) a través de [nodos](/nodos/) en un [cluster](/cluster/). Agrupa muchos componentes de infraestructura, en ocasiones referidos como "primitivos", como una instancia de una aplicación, balanceadores de carga, almacenamiento persistente y otros, de forma que al unirse son piezas que juntas forman parte de una aplicación.
+
+Kubernetes habilita la automatización y extensibilidad, permitiendo a los usuarios desplegar sus aplicaciones de una manera declarativa y reproducible. Los productos de Software en el Ecosistema de Kubernetes toman ventaja de la automatización para extender la [API](/api/) de Kubernetes. Lo que les permite mejorar la automatización en Kubernetes y hacer sus aplicaciones más accesibles a usuarios experimentados de Kubernetes.
+
+## Problema que aborda
+La automatización de infrastructura y administración de configuración declarativa han sido conceptos muy importantes por un largo tiempo, y han sumado más importancia desde que el [cloud computing](/cloud_computing/) ha ganado popularidad. La demanda de recursos de cómputo incrementa día con día y las organizaciones sienten la presión de proveer más capacidad operativa, con menos ingenieros. Para cumplir con esta demanda nuevas tecnologías y metodologías de trabajo deben ser creadas.
+
+A su vez también otros conceptos como lo son los [contenedores](/contenedores/) también han ganado mucha atención. Compañías que estaban ocupadas automatizando infraestructura tradicional ahora necesitan mecanismos para automatizar la configuración y el despliegue de sus contenedores.
```

Comment:
[en]
> Additionally, the rise of cloud computing was paired with [containerization](https://deploy-preview-724--cncfglossary.netlify.app/containerization/) and organizations that were busy automating more traditional infrastructure needed a mechanism to automate the configuration and deployment of their containers.

In English version, there exists a term "containerization", but Google translator says that 'contenedores' here corresponds to 'container', not 'containerization'

Please check! 😊

My suggestion (this might be not right): 
```suggestion
A su vez también otros conceptos como lo son los [contenerización](/containerization/) también han ganado mucha atención. Compañías que estaban ocupadas automatizando infraestructura tradicional ahora necesitan mecanismos para automatizar la configuración y el despliegue de sus contenedores.
```